src/gui/main_window.py

```diff
@@ -191,6 +195,15 @@ def __init__(self, parent=None):
                             'widget': CsvExport(),
                             'help': help_attribute_table()
                         }
+                    },
+
+                    {#ajoute par Rachel Goree 30/05/2017 
```

Comment:
Like here too, error `E261 at least two spaces before inline comment` and `E262 inline comment should start with '# '`

So it has to be `{  # ajoute par Rachel Goree 30/05/2017`